### PR TITLE
[ID] Add `Beatmap Spotlight Curators`

### PR DESCRIPTION
**Pull request by maxuskazuma**
right now indonesia version doesn't have this new page being updated, so we need to have new one.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+tags:
+    - BSC
+    - beatmap spotlight curator
+---
+
+# Beatmap Spotlight Curators
+
+**Beatmap Spotlight Curators** (***BSC***) adalah grup yang membantu memilih map berkualitas tinggi untuk setiap season dari [Beatmap Spotlights](/wiki/Beatmap_Spotlights).
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:9]:
```suggestion
**Beatmap Spotlight Curators** (***BSC***) adalah sebuah grup yang membantu memilih map-map berkualitas tinggi untuk setiap musim dari [Beatmap Spotlights](/wiki/Beatmap_Spotlights).
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Menjadi kurator
+
+Pada umumnya, kurator baru disarankan oleh anggota tim dari setiap game mode untuk memenuhi kriteria tertentu. Sugesti tersebut dievaluasi oleh Manajer projek, pemimpin team, dan anggota kurator sendiri.
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:13]:
```suggestion
Pada umumnya, menjadi seorang kurator baru perlu disarankan oleh anggota tim dari setiap mode permainan untuk memenuhi kriteria tertentu. Saran tersebut akan dievaluasi oleh Manajer projek, pemimpin tim, dan anggota kurator sendiri.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Selain itu, mungkin juga akan dibuka pendaftaran untuk beberapa gamemode yang diumumkan untuk musim tertentu.
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:15]:
```suggestion
Selain itu, kemungkinan pendaftaran akan dibuka untuk beberapa mode permainan yang diumumkan untuk musim tertentu.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Semua pendaftar akan dinilai berdasarkan keahlian sebagai pemain, mapper, modder, dan tournament mappool selector untuk masing-masing game mode. Pendaftar tidak perlu untuk ahli di semua bidang ini untuk bisa dipilih, tapi mempunyai pengalaman yang luas membantu. Tujuannya adalah untuk memiliki anggota kurator dengan campuran dari berbagai jenis skill level, pengalaman, and keahlian. Akan tetapi, jumlah kurator pada saat ini dibuat kecil and terbatas karena proyek masih berkembang.
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:17]:
```suggestion
Semua pendaftar akan dinilai berdasarkan keahlian-keahlian yang dimiliki sebagai pemain, mapper, modder, dan tournament mappool selector untuk masing-masing mode permainan. Pendaftar tidak perlu menjadi ahli di semua bidang ini untuk bisa dipilih, tapi mempunyai pengalaman yang luas membantu. Tujuannya adalah untuk memiliki anggota kurator dengan campuran dari berbagai jenis skill level, pengalaman, dan keahlian. Akan tetapi, jumlah kurator pada saat ini dibuat kecil dan terbatas karena proyek masih berkembang.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+## Curators
+
+Proyek Beatmap Spotlights dijalankan oleh berbagai anggota komunitas di semua mode game, dipimpin oleh pimpinan proyek khusus. [Halaman Grup Beatmap Spotlight Curators](https://osu.ppy.sh/groups/48) mencantumkan nama semua anggota.
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:21]:
```suggestion
Proyek Beatmap Spotlights dijalankan oleh berbagai anggota komunitas di semua mode permainan yang dipimpin oleh pimpinan proyek khusus. [Halaman Grup Beatmap Spotlight Curators](https://osu.ppy.sh/groups/48) mencantumkan nama semua anggota.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+| Role | Members |
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:23]:
```suggestion
| Peran | Anggota |
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+| :-- | :-- |
+| Pimpinan proyek | ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) |
+| Proyek manajer | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) |
+
+Tim kurator khusus mengumpulkan setiap rilis Beatmap Spotlights. Pemimpin tim ditandai dengan huruf tebal.
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:28]:
```suggestion
Tim kurator khusus berdedikasi mengumpulkan setiap rilis dari Beatmap Spotlights. Pemimpin tim ditandai dengan huruf tebal.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+### osu! curators
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:30]:
```suggestion
### kurator osu!
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+- ::{ flag=FI }:: **[Nowaie](https://osu.ppy.sh/users/5428909)**
+- ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251)
+- ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334)
+- ::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207)
+- ::{ flag=FI }:: [Lefafel](https://osu.ppy.sh/users/2295850)
+- ::{ flag=AT }:: [Omgforz](https://osu.ppy.sh/users/578943)
+- ::{ flag=MX }:: [Riot](https://osu.ppy.sh/users/4256461)
+- ::{ flag=PL }:: [Zelq](https://osu.ppy.sh/users/8953955)
+
+### osu!taiko curators
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:41]:
```suggestion
### kurator osu!taiko
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+- ::{ flag=TN }:: **[Hivie](https://osu.ppy.sh/users/14102976)**
+- ::{ flag=MY }:: [[Zeth]](https://osu.ppy.sh/users/9912966)
+- ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864)
+- ::{ flag=GB }:: [Horiiizon](https://osu.ppy.sh/users/8071438)
+- ::{ flag=US }:: [Nifty](https://osu.ppy.sh/users/4956097)
+- ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099)
+- ::{ flag=JP }:: [uone](https://osu.ppy.sh/users/5321719)
+
+### osu!catch curators
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:51]:
```suggestion
### kurator osu!catch
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+- ::{ flag=CA }:: **[SadEgg](https://osu.ppy.sh/users/10278243)**
+- ::{ flag=RU }:: [Kimitakari](https://osu.ppy.sh/users/4741164)
+- ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099)
+- ::{ flag=CA }:: [WadBot](https://osu.ppy.sh/users/14571181)
+- ::{ flag=US }:: [wonjae](https://osu.ppy.sh/users/5032045)
+- ::{ flag=CA }:: [wwwww](https://osu.ppy.sh/users/8434466)
+
+### osu!mania curators
```
Comment (mousesalt) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:60]:
```suggestion
### kurator osu!mania
```

**Review by mousesalt — APPROVED** [review]
approved

**Review by mousesalt — APPROVED** [review]
second attempt

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+## Curators
+
+Projek Beatmap Spotlights dijalankan oleh berbagai anggota komunitas di semua mode permainan yang dipimpin oleh pimpinan projek khusus. [Halaman Grup Beatmap Spotlight Curators](https://osu.ppy.sh/groups/48) mencantumkan nama semua anggota.
+
+| Peran | Anggota |
+| :-- | :-- |
+| Pimpinan projek | ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) |
+| Projek manajer | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) |
+
+Tim kurator khusus berdedikasi mengumpulkan setiap rilis dari Beatmap Spotlights. Pemimpin tim ditandai dengan huruf tebal.
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:28]:
```suggestion
Tim kurator khusus disusun untuk setiap rilis Beatmap Spotlights. Pemimpin tim ditandai dengan huruf tebal.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+### kurator osu!
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:30]:
```suggestion
### Kurator osu!
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+- ::{ flag=FI }:: **[Nowaie](https://osu.ppy.sh/users/5428909)**
+- ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251)
+- ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334)
+- ::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207)
+- ::{ flag=FI }:: [Lefafel](https://osu.ppy.sh/users/2295850)
+- ::{ flag=AT }:: [Omgforz](https://osu.ppy.sh/users/578943)
+- ::{ flag=MX }:: [Riot](https://osu.ppy.sh/users/4256461)
+- ::{ flag=PL }:: [Zelq](https://osu.ppy.sh/users/8953955)
+
+### kurator osu!taiko
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:41]:
```suggestion
### Kurator osu!taiko
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+- ::{ flag=TN }:: **[Hivie](https://osu.ppy.sh/users/14102976)**
+- ::{ flag=MY }:: [[Zeth]](https://osu.ppy.sh/users/9912966)
+- ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864)
+- ::{ flag=GB }:: [Horiiizon](https://osu.ppy.sh/users/8071438)
+- ::{ flag=US }:: [Nifty](https://osu.ppy.sh/users/4956097)
+- ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099)
+- ::{ flag=JP }:: [uone](https://osu.ppy.sh/users/5321719)
+
+### kurator osu!catch
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:51]:
```suggestion
### Kurator osu!catch
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+- ::{ flag=CA }:: **[SadEgg](https://osu.ppy.sh/users/10278243)**
+- ::{ flag=RU }:: [Kimitakari](https://osu.ppy.sh/users/4741164)
+- ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099)
+- ::{ flag=CA }:: [WadBot](https://osu.ppy.sh/users/14571181)
+- ::{ flag=US }:: [wonjae](https://osu.ppy.sh/users/5032045)
+- ::{ flag=CA }:: [wwwww](https://osu.ppy.sh/users/8434466)
+
+### kurator osu!mania
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:60]:
```suggestion
### Kurator osu!mania
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+    - beatmap spotlight curator
+---
+
+# Beatmap Spotlight Curators
+
+**Beatmap Spotlight Curators** (***BSC***) adalah sebuah grup yang membantu memilih map-map berkualitas tinggi untuk setiap musim dari [Beatmap Spotlights](/wiki/Beatmap_Spotlights).
+
+## Menjadi kurator
+
+Pada umumnya, menjadi seorang kurator baru perlu disarankan oleh anggota tim dari setiap mode permainan untuk memenuhi kriteria tertentu. Saran tersebut akan dievaluasi oleh Manajer projek, pemimpin tim, dan anggota kurator sendiri.
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:13]:
```suggestion
Pada umumnya, kurator baru direkrut berdasarkan rekomendasi yang diberikan oleh anggota tim dari setiap mode permainan untuk memenuhi kriteria tertentu. Rekomendasi tersebut akan dievaluasi oleh manajer proyek, pemimpin tim, dan anggota kurator itu sendiri.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+Selain itu, kemungkinan pendaftaran akan dibuka untuk beberapa mode permainan yang diumumkan untuk musim tertentu.
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:15]:
```suggestion
Selain itu, pendaftaran dapat dibuka untuk beberapa mode permainan yang diumumkan untuk musim tertentu.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+Semua pendaftar akan dinilai berdasarkan keahlian-keahlian yang dimiliki sebagai pemain, mapper, modder, dan tournament mappool selector untuk masing-masing mode permainan. Pendaftar tidak perlu menjadi ahli di semua bidang ini untuk bisa dipilih, tapi mempunyai pengalaman yang luas membantu. Tujuannya adalah untuk memiliki anggota kurator dengan campuran dari berbagai jenis skill level, pengalaman, dan keahlian. Akan tetapi, jumlah kurator pada saat ini dibuat kecil dan terbatas karena projek yang masih berkembang.
```
Comment (ZeroPyrozen) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/id.md:17]:
```suggestion
Semua pendaftar akan dinilai berdasarkan keahlian-keahlian yang dimiliki sebagai pemain, mapper, modder, dan tournament mappool selector untuk masing-masing mode permainan. Pendaftar tidak perlu menjadi ahli di semua bidang ini untuk bisa dipilih, tetapi mempunyai pengalaman yang luas dapat membantu. Tujuannya adalah untuk memiliki anggota kurator dengan campuran dari berbagai jenis skill level, pengalaman, dan keahlian. Akan tetapi, jumlah kurator pada saat ini dibuat kecil dan terbatas karena proyek yang masih berkembang.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — DISMISSED** [review]

**Review by mousesalt — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]
I've combined your changes into 3 commits instead of 27, but please follow these best practices in your future PRs:
- make [commits with meaningful descriptions](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits);
- for a single logically scoped change, such as applying a review, make a single commit (`Apply review from <someone>` or so), as opposed to 10 small commits titled `Update Path/To/Article/en.md`

**Pull request merged**